app/helpers/admin/images_helper.rb

```diff
@@ -10,12 +10,6 @@ def update_image(param_path)
   end
 
   def add_image(param_path)
```

Comment:
I think I ended up needing to write this method all funky like this with two weird methods for the edit functionality later on, but I could be completely wrong. either way, much cleaner now. nice work! :sparkles: